mypage/index.html

```diff
@@ -80,7 +80,101 @@ <h1 class="mypage-content__title">내가 등록한 강의</h1>
+                      </button>
+                      <button
+                        type="button"
+                        class="button button--ghost button--danger"
+                      >
+                        삭제하기
+                      </button>
+                    </td>
+                  </tr>
+                  <tr class="course-row">
+                    <td>
+                      <div class="table-thumb">
+                        <img
+                          src="https://images.unsplash.com/photo-1526481280695-3c4696001f81?auto=format&fit=crop&w=640&q=80"
+                        />
+                      </div>
+                    </td>
+                    <td>
+                      <div class="table-title">
+                        <strong>UX/UI 디자인 워크샵</strong>
+                        <!-- <span>비공개 전환 · 2023.10.01</span> -->
+                      </div>
+                    </td>
+                    <!-- <td><span class="badge badge--private">비공개</span></td> -->
+                    <td>디자인</td>
+                    <td class="table-actions">
+                      <button
+                        type="button"
+                        class="button button--secondary modal-toggle-btn"
+                        data-modal-mode="course-edit">
+                        수정하기
+                      </button>
+                      <button
+                        type="button"
+                        class="button button--ghost button--danger"
+                      >
+                        삭제하기
+                      </button>
+                    </td>
+                  </tr>
+                  <tr class="course-row">
+                    <td>
+                      <div class="table-thumb">
+                        <img
+                          src="https://images.unsplash.com/photo-1526481280695-3c4696001f81?auto=format&fit=crop&w=640&q=80"
+                        />
+                      </div>
+                    </td>
+                    <td>
+                      <div class="table-title">
+                        <strong>JavaScript 실전 프로젝트</strong>
+                        <!-- <span>일시 중지 · 2023.09.20</span> -->
+                      </div>
+                    </td>
+                    <!-- <td><span class="badge badge--private">비공개</span></td> -->
+                    <td>프로그래밍</td>
+                    <td class="table-actions">
+                      <button
+                        type="button"
+                        class="button button--secondary modal-toggle-btn"
+                        data-modal-mode="course-edit">
+                        수정하기
+                      </button>
+                      <button
+                        type="button"
+                        class="button button--ghost button--danger"
+                      >
+                        삭제하기
+                      </button>
+                    </td>
+                  </tr>
+                </tbody>
```

Comment:
```suggestion
                <tbody class="course-table__body">
                </tbody>
```
삭제된 기존 데이터가 다시 반영이 되었네요! 
<tbody class="course-table__body"></tbody>
만 남겨놓고 제거해야 할 듯 합니다!